home/home3.1/InverseTest.java

```diff
@@ -0,0 +1,37 @@
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+public class InverseTest {
+
+    @Test
+    public void PasitiveArguments() {
+        Inverse inverse = new Inverse();
+        double result = inverse.calculateInverse(2, 3, 4);
+        double expected=1.0/24.0;
+        assertEquals(expected, result, "iverse to the 24 should be equal to 1/24");
```

Comment:
There is no need to have some message especially with mistype "iverse"
Copy paste in comments may be also a problem in case of code changes as you need to support both copies(implementation and comment). Therefore comments should provide some extra information not duplicate the things obvious from code itself.